src/adapters/marketplace/v00.js

```diff
@@ -225,7 +225,7 @@ class V00_MarkeplaceAdapter {
       } else if (event.event === 'OfferDisputed') {
         offers[event.returnValues.offerID] = { status: 'disputed', event }
       } else if (event.event === 'OfferRuling') {
-        offers[event.returnValues.offerID] = { status: 'resolved', event }
+        offers[event.returnValues.offerID] = { status: 'ruled', event }
```

Comment:
The grammar is a bit odd, but I'd rather stay consistent with having js status' match the event names, and have this as `ruling` rather than `ruled`.